support-frontend/assets/helpers/abTests/abtest.ts

```diff
@@ -154,8 +163,10 @@ function getParticipations(
 	mvtId: number,
 	country: IsoCountry,
 	countryGroupId: CountryGroupId,
+	path: string,
```

Comment:
To make this function easier to test, we now pass in a `path` instead of using `window.location.pathname` directly